spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/EventProcessorProperties.java

```diff
@@ -123,8 +123,13 @@ public static class ProcessorSettings {
         private String sequencingPolicy;
 
         /**
-         * Returns the name of the bean that should be used as source for Event Messages. If not provided, the {@link
-         * org.axonframework.eventhandling.EventBus} is used as source.
+         * The sequenced dead letter queue settings to be used for this processing group.
```

Comment:
```suggestion
         * The {@link org.axonframework.messaging.deadletter.SequencedDeadLetterQueue} settings that will be used for this processing group.
```

Reply:
I think it's fair to link to the interface in the other "sequenced dead letter queue" references you've added to the JavaDoc.